python/docs/_scripts/builder.py

```diff
@@ -6,7 +6,7 @@
 from typing import Any, Union
 
 from pydantic import ValidationError
-from quartodoc import blueprint, collect, layout, preview
+from quartodoc import blueprint, collect, layout  # , preview
```

Comment:
delete?

Reply:
I wanted to keep this since its super helpful for debugging... so I don't forget it exists.  I can move it to a separate line and add a comment about it.